cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -273,8 +275,9 @@ def send_tarball(tarball, size):
     """
     Renders a tarball to response, for use when sending a tar.gz file to the user.
     """
-    wrapper = FileWrapper(tarball)
-    response = HttpResponse(wrapper, content_type='application/x-tgz')
+    chunk_size = 4096
```

Comment:
Instead of this constant being hard-coded here, could you turn it into a setting so that it could be updated without needing a code rollout. Or minimally pull it out to module-level and provide a comment as to why it's set to the value of 4096.

Reply:
Sure.